lab_2_retrieval_w_bm25/main.py

```diff
@@ -3,9 +3,10 @@
 
 Text retrieval with BM25
```

Comment:
you should have only 3 changed file, now you have 6(!)
![image](https://github.com/user-attachments/assets/e4260433-7bb8-43d3-b978-c1499cc0083e)
Fix that

Reply:
i trird to fix it, i promise i will do it